src/Digdir.Domain.Dialogporten.Application/ApplicationExtensions.cs

```diff
@@ -19,6 +19,12 @@ public static IServiceCollection AddApplication(this IServiceCollection services
         ArgumentNullException.ThrowIfNull(configuration);
         var thisAssembly = Assembly.GetExecutingAssembly();
 
+        // FIXME! Is this the best way to do this?
```

Comment:
Virker litt grisete å skulle bygge IConfiguration på nytt her for å få inn secrets fra Application ..?

Reply:
Ja, det er grisete. Ser at du gjør det fordi du bruker det i tool generatoren din. Det er kun runtime prosjektets secrets som lastes av dotnets configuration default. Derfor har vi samme secret id på alle presentasjonslagene våre (api og service). I tillegg har også infrastruktur den samme secret iden. Dette for å kunne bruke ef cli for å lage administrere migrasjoner, som krever connection stringen til postgre. 

Jeg ville brukt samme secret id som alle andre presentasjonslag i toolen din også og administrert user secrets på "normal" måte. Men har du vurdert å legge dette inn som en del av oppstarten lokalt slik at nye utviklere ikke trenger enda et steg for å komme i gang med dialogporten (DX)? 😁

Reply:
Nei, Tool.Dialogporten.Ed25519KeyPairGenerator har ingen dependencies (foruten NSec)

Dialogtokens er applikasjosnfunksjonalitet og er rent foretningslogisk, så jeg oppfatter ikke det som infrastruktur. Konfigurasjonen av dette bør følgelig ligge under Application-"navnerommet" - og her trenger vi å få inn nøkkelparene som benyttes (som da er noe som genereres ved første gangs deploy av et miljø, og legges inn i keyvault.) 

Men ser poenget med å bare bruke samme secretid, skal endre det. Mht DX så er det by default lagt inn et flagg i LocalDevelopmentSettings som da kortslutter (i praksis deaktiverer) hele greia, så ut av boksen er det ikke behov for å gjøre noe for å få opp miljøet. Men om du vil teste/utvikle noe rundt dialogtokens, er det behov for å konfigurere opp dette - og uten det verktøyet for å generere nøkkelpar er det en smule knotete. Det verktøyet kan også justeres til å kunne brukes ifm miljøoppsett.

Reply:
Fjernet nå, har samme secretsid som i webapi